packages/web3-auth/src/SocialLogin.tsx

```diff
@@ -56,7 +55,7 @@ class SocialLogin {
   }
 
   async init(
-    chainId: string,
+    chainId: string = '0x1',
```

Comment:
instead of using '0x1' let's use a constant which can be in something like a config file. Refrain from using magic numbers/strings.